### PR TITLE
Use LimitWordCount instead of LimitWordCountXML

### PR DESCRIPTION
**Pull request by martinduparc**
Thanks for contributing, you're awesome! :star:
Please describe expected and observed behaviour, and what you're fixing.
For visual fixes, please include tested browsers and screenshots.
Search for related existing issues and link to them if possible.
Please read https://docs.silverstripe.org/en/contributing/code/

### CODE REVIEW

**Comment by dhensby** [comment]
Hi @martinduparc, please can you provide some reasoning as to why you think this change is needed?

**Comment by martinduparc** [comment]
The method was removed in SS4. See `Removed String::LimitWordCountXML(). Use LimitWordCount() instead.` in https://github.com/silverstripe/silverstripe-framework/blob/4.0/docs/en/04_Changelogs/4.0.0.md#orm-overview-orm

**Comment by dhensby** [comment]
well that's a pretty good justification :P

**Comment by dhensby** [comment]
Should it be `$Content.LimitWordCount.XML`, then?

**Comment by martinduparc** [comment]
`.XML` or its alias `.HTML` (http://api.silverstripe.org/4/SilverStripe/ORM/FieldType/DBString.html#method_HTML)

**Comment by dhensby** [comment]
`.XML` - it's the canonical method and it's what is more familiar.

**Comment by martinduparc** [comment]
Sounds good. Thanks for the quick response!

File: docs/en/01_Tutorials/04_Site_Search.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ class.
                 $Title
                 <% end_if %>
             </a>
-            <p>$Content.LimitWordCountXML</p>
+            <p>$Content.LimitWordCount</p>
```
Comment (dhensby) [inline docs/en/01_Tutorials/04_Site_Search.md]:
```diff
-<p>$Content.LimitWordCount</p>
+<p>$Content.LimitWordCount.XML</p>
```

**Comment by martinduparc** [comment]
Thanks @robbieaverill

**Comment by robbieaverill** [comment]
Thank you for the contribution!

**Pull request merged**